### PR TITLE
Add Stylus Icon

### LINKED ISSUE #10719: Request: Stylus
### Brand Name

Stylus

### Website

https://github.com/openstyles/stylus/

### Popularity Metric

Has 5k stars (https://github.com/openstyles/stylus), consistent upward trajectory (https://star-history.com/#openstyles/stylus). I'd also point out that it has 700k users with a 4.5 star rating across 1k reviews at https://chromewebstore.google.com/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne - plus another 100k users with a 4.7 star rating across 1k reviews at https://addons.mozilla.org/en-US/firefox/addon/styl-us/.

### Official Resources for Icon and Color

Couldn't locate an SVG version, PNG version: https://github.com/openstyles/stylus/blob/master/images/icon/128.png.

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by EricRisher**
image

**Issue:** closes #10719  

**Popularity metric:** The github has a currently 5026 stars and a upward trajectory. See 
https://star-history.com/#openstyles/stylus&Date
 

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

#### Name
As it appears on their website and uncenter's suggestion to add extension after because stylus icon is already taken.
- [Stylus](https://add0n.com/stylus.html)

#### Icon 
Derived from the their official github svg icon and removed styling so that logo is monochromatic https://gist.github.com/pabli24/85c406ebfdb840cb0450e029b28948b8#file-stylus-logo-svg

#### Color
Extracted from original svg icon color 
- `#285959`

### CODE REVIEW

**Comment by uncenter** [comment]
It looks like you've made a new branch based off of your Quizlet branch. That's why you can see your Quizlet commits above the Stylus commit. Do you mind creating a new branch based on the `develop` branch and readding the Stylus extension SVG and data?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -14721,6 +14726,11 @@
             "hex": "333333",
             "source": "https://github.com/stylus/stylus-lang.com/blob/c833bf697e39e1174c7c6e679e0e5a23d0baeb90/img/stylus-logo.svg"
         },
+        {
+            "title": "Stylus Extension",
```
Comment (adamrusted) [inline _data/simple-icons.json:14730]:
```suggestion
            "title": "Stylus",
            "slug": "stylus_extension",
```

We can utilise the slug field for this purpose (see example in [this section](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#optional-data) of our CONTRIBUTING.md)

**Review by adamrusted — CHANGES_REQUESTED** [review]
Please try to keep your PRs to only one icon at a time. Could you look to split Quizlet into it's own PR please @EricRisher?

**Comment by EricRisher** [comment]
> Please try to keep your PRs to only one icon at a time. Could you look to split Quizlet into it's own PR please @EricRisher?

Hey Adam I'm still really new to contributing to open source projects, could you please inform me on how to split the PRs?

**Comment by adamrusted** [comment]
Sure - I'd just revert the addition of the `Quizlet` changes in this branch - that'd take us back to just having the one icon in this one!

**Comment by adamrusted** [comment]
I've removed Quizlet, and had a go at doing something different with the paths. This one is a real pain to work with - so kudos for giving it a shot @EricRisher! Could another @simple-icons/pr-reviewers take a look at this now I've contributed?

File: icons/stylus_extension.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...4 2.343v7.407s-.094.843-1.5.843v1.125c1.406 0 1.5.844 1.5.844v7.219c0 1.312 1.031 2.344 2.344 2.344h14.062c1.313 0 2.344-1.032 2.344-2.344V13.5s.094-.844 1.5-.844v-1.125c-1.406 0-1.5-.843-1.5-.843V3.281c0-1.312-1.031-2.343-2.344-2.343H4.969ZM20.375 13.5v7.219c0 .752-.591 1.344-1.344 1.344H4.969c-.753 0-1.344-.592-1.344-1.344v-7.147c.004-.061.001-.143-.013-.231-.012-.082-.077-.52-.483-.949a2.1008 2.1008 0 0 0-.348-.298c.142-.097.256-.2.348-.298.406-.429.471-.868.483-.949a.2912.2912 0 0 0 .006-.045.8768.8768 0 0 0 .007-.115V3.281c0-.752.591-1.343 1.344-1.343h14.062c.753 0 1.344.591 1.344 1.343v7.407l.007.114s.002.018.006.045c.012.081.077.52.483.949.092.098.206.201.348.298-.142.097-.256.2-.348.298-.406.429-.471.867-.483.949-.004.026-.006.044-.006.044s-.007.667-.007.115ZM12.844 2.953c-4.125 0-6.656 2.297-6.656 5.297 0 2.531 1.921 4.125 4.828 5.203 2.109.703 2.953 1.313 2.953 2.438 0 1.125-.938 1.875-2.766 1.875-1.687 0-3.375-.516-4.406-1.078L6 19.922c.984.562 2.953 1.125 4.969 1.125 4.781 0 7.031-2.531 7.031-5.438 0-2.484-1.406-4.125-4.5-5.25-2.297-.843-3.281-1.359-3.281-2.484 0-.891.89-1.687 2.578-1.687 1.687 0 2.953.515 3.609.843l.891-3.187c-1.031-.469-2.484-.891-4.453-.891Z"/></svg>
```
Comment (service-paradis) [inline icons/stylus_extension.svg:1]: Looking at the source, I think I would opt for the following treatment 
image

Where did you see the current one @adamrusted?

**Review by service-paradis — CHANGES_REQUESTED** [review]

File: icons/stylus_extension.svg (hunk 2 above)
Comment (adamrusted) [reply icons/stylus_extension.svg:1]: I created it myself based on the colour logo - the treatment you've shown above does _not_ display well at the smaller sizes our project are generally used for - so I tried to counter that by just having the inlay.

**Comment by adamrusted** [comment]
Closing as abandoned. No updates since April.

**Pull request closed without merging**